install/installer/pkg/components/registry-facade/podsecuritypolicy.go

```diff
@@ -38,8 +38,8 @@ func podsecuritypolicy(ctx *common.RenderContext) ([]runtime.Object, error) {
 			HostIPC:     false,
 			HostPID:     false,
 			HostPorts: []v1beta1.HostPortRange{{
-				Min: 30000,
-				Max: 33000,
+				Min: 20000,
+				Max: 20000,
```

Comment:
Presume this is fine with the min/max being the same value?

Reply:
We are exposing only one port. Also this was changed after feedback from @sagor999 https://github.com/gitpod-io/gitpod/pull/8580#discussion_r819019732